lib/flatpak-installation.c

```diff
@@ -2041,3 +2041,84 @@ flatpak_installation_list_installed_related_refs_sync (FlatpakInstallation *self
 
   return g_steal_pointer (&refs);
 }
+
+/**
+ * flatpak_installation_remove_local_ref_sync
+ * @self: a #FlatpakInstallation
+ * @remote_name: the name of the remote
+ * @ref: the ref
+ * @cancellable: (nullable): a #GCancellable
+ * @error: return location for a #GError
+ *
+ * Remove the OSTree ref given by @remote_name:@ref from the local flatpak
+ * repository. The next time the underlying OSTree repo is pruned, objects
+ * which were attached to that ref will be removed. This is useful if you
+ * pulled a flatpak ref using flatpak_installation_install_full and
+ * specified FLATPAK_INSTALL_FLAGS_NO_DEPLOY but then decided not to
```

Comment:
Nitpick: Prefix `FLATPAK_INSTALL_FLAGS_NO_DEPLOY` with `%` to get gtk-doc to format it appropriately.